File: src/admin-panel/src/app/models/UserManagementModel.ts
import { Role } from "./roles";

export interface UserManagementModel {
    id: string;
    email: string;
    fullName: string;
    roles: Role[];
    imageUrl?: string;
    course?: number;
    picture: string;
    lastLoginDateTimeUtc?: string;
    isActive: boolean;
    faculty: string;
}

File: src/api/EProfspilka.Core/Responses/UserManagementModel.cs
using EProfspilka.Core.Enumerations;

namespace EProfspilka.Core.Responses;

public class UserManagementModel
{
    public Guid Id { get; set; }
    public string FullName { get; set; }
    public string Email { get; set; }

    public bool IsActive { get; set; }
    public DateTime? LastLoginDateTimeUtc { get; set; }

    public string Faculty { get; set; }
    public int? Course { get; set; }
    public string Picture { get; set; }
    public List<Role> Roles { get; set; }
}